src/components/TransactionHistoryRows/utils.ts

```diff
@@ -105,3 +106,28 @@ export const makeAmountOrDefault = (
   // All above infinite/revoke checks failed, this is an exact approval
   return `${approvedAmount} ${approvedAsset.symbol}`
 }
+
+export const useIsOverflow = (
```

Comment:
*not* nitpicky, move me to a better home e.g `hooks/useIsOverflow.ts` or similar @reallybeard